hcs-sxc-java-examples/hcs-sxc-java-settlement-demo/config/docker-compose.yml

```diff
@@ -0,0 +1,121 @@
+version: '3.3'
```

Comment:
docker-compose shouldn't really be checked in. Should be .gitignored

Reply:
done, deleted and kept local